docs/coding.md

```diff
@@ -76,6 +76,26 @@ Follow these instructions to set up Keycloak for Authentication/Authorization.
    - username: `developer`, password: `developer`
    - username: `admin`, password: `admin`
 
+> NOTE: If you'd like to use demo data users and have them automatically imported in keycloak, run the keycloak init script as below: 
+```shell
+USE_DEMO_DATA=false yarn keycloak:init
+``` 
+
+This command will create the following usernames:
+
+```log
+hzaub	
+kudi
+msash
+tgers
+tmaure
+ukon
+wtrocki
```

Comment:
LOL - this is accidental name but like it. I would add yours as well :D

Reply:
:-)